scripts/upstream.sh

```diff
@@ -91,14 +91,12 @@ start_rebase() {
 
   for patch in ../patches/*.patch; do
     echo "Applying $patch"
-    if ! git apply --3way "$patch"; then
+    if ! git am --3way "$patch"; then
       echo
       echo "Failed to apply patch. Please run 'make upstream.rebase FROM=$TAG' where '$TAG' allows the patch set to apply cleanly"
       echo
       exit 1
     fi
-    patch=${patch#../patches/*-}
```

Comment:
What's the new equivalent of this?

Reply:
This extracted the filename of the patch to use as the commit message in the line below. `git am` preserves the existing commit message (subject & body).

See: https://github.com/pulumi/ci-mgmt/pull/469